views/engineering_garage/checkout_user.erb

```diff
@@ -0,0 +1,247 @@
+										<% end %>
+										<% if project[:last_checked_in] != nil %>
+											<%= project[:last_checked_in].in_time_zone.strftime("%b %-d, %Y %-I:%M%P") %>
+										<% end %>
+									<% end %>
+
+									</td>
+                  <td class="dcf-d-flex dcf-flex-row-rev dcf-flex-shrink-1 dcf-gap-2">
+										<button class="dcf-btn-toggle-modal dcf-btn dcf-btn-primary" data-toggles-modal="delete-<%= project[:id] %>" type="button">Delete</button>
+										<div class="dcf-modal" id="delete-<%= project[:id] %>" hidden>
+											<div class="dcf-modal-wrapper">
+												<div class="dcf-modal-header">
+													<h2>Delete Project</h2>
+													<button class="dcf-btn-close-modal">Close</button>
+												</div>
+												<div class="dcf-modal-content">
+													<form class="dcf-form dcf-grid" action="/checkout/project_delete/?nuid=<%= nuid %>&bin_id=<%= project[:bin_id] %>" method="POST" enctype="application/x-www-form-urlencoded">
+														<div class="dcf-col-50%-start">
+															<p id="project_name"><strong>Name:</strong> <%= project[:title] %></p>
+															<p id="bin_id"><strong>Location:</strong> <%= project[:bin_id] %></p>
+															<p><strong>Description:</strong> <%= project[:description] %></p>
+															<div class="dcf-form-controls-inline dcf-gap-2">
+																<button class="dcf-btn dcf-btn-primary" type="submit">Delete</button>
+															</div>
+														</div>
+													</form>
+												</div>
+											</div>
+										</div>
+									  <% if project[:last_checked_out] == nil || project[:last_checked_out] <= project[:last_checked_in] %>
+											<button class="dcf-btn-toggle-modal dcf-btn dcf-btn-primary" data-toggles-modal="checkout-<%= project[:id] %>" type="button">Check Out</button>
+											<div class="dcf-modal" id="checkout-<%= project[:id] %>" hidden>
+												<div class="dcf-modal-wrapper">
+													<div class="dcf-modal-header">
+														<h2>Check Out Project</h2>
+														<button class="dcf-btn-close-modal">Close</button>
+													</div>
+													<div class="dcf-modal-content">
+														<form class="dcf-form dcf-grid" action="/checkout/project_checkout/?nuid=<%= nuid %>&bin_id=<%= project[:bin_id] %>" method="POST" enctype="application/x-www-form-urlencoded">
+															<div class="dcf-col-50%-start">
+																<p id="project_name"><strong>Name:</strong> <%= project[:title] %></p>
+																<p id="bin_id"><strong>Location:</strong> <%= project[:bin_id] %></p>
+																<p><strong>Description:</strong> <%= project[:description] %></p>
+																<div class="dcf-form-controls-inline dcf-gap-2">
+																	<button class="dcf-btn dcf-btn-primary" type="submit">Check Out</button>
+																</div>
+															</div>
+															<div class="dcf-col-50%-end dcf-h-fixed-100% dcf-overflow-y-auto">
```

Comment:
Remove the style `dcf-h-fixed-100%` and replace with an inline style for `max-height: 30rem;`